### ISSUE TITLE
Doesnt work for virtual machine, followed all steps for VM 

### ISSUE THREAD

**Issue by piush-terminalwitchcraft** [Axarva/dotfiles-2.0#167]
![image](https://user-images.githubusercontent.com/76522725/188120198-706aebf1-5582-4871-a727-68927268681f.png)
It gets stuck after login , doesnt proceed further , new to arch linux help would be appreciated :)

**Comment by Axarva** [comment]
Please include your ~/.xinitrc too.

**Comment by piush-terminalwitchcraft** [comment]
![image](https://user-images.githubusercontent.com/76522725/188123450-818c6b18-2edb-465c-8a77-44efa9fdaad6.png)
Its just a command 😅I really dont what is wrong

**Comment by Axarva** [comment]
I'm not really sure what's wrong if you followed the steps in `VM` too. Are you sure xmonad is installed correctly?

**Comment by piush-terminalwitchcraft** [comment]
Yes,  i have referred to one of the youtube videos, there was a slight difference at the end, it shows xmonad is forced to recompile then it is showing success

**Comment by Axarva** [comment]
You *could* try that. On a tty, try the following command: `xmonad --recompile`

**Comment by piush-terminalwitchcraft** [comment]
![image](https://user-images.githubusercontent.com/76522725/188127543-de118597-f13b-4e99-8156-d342a19d18ed.png)
Got something strange. But it is showing success

**Comment by Axarva** [comment]
Now try `startx`

**Comment by piush-terminalwitchcraft** [comment]
![image](https://user-images.githubusercontent.com/76522725/188128036-af1730be-795d-4ae3-840b-b8ed2a297c98.png)
Didnt worked.... I guess there must be some mistake in installing packages

**Comment by piush-terminalwitchcraft** [comment]
![image](https://user-images.githubusercontent.com/76522725/188130700-b22ac95b-3212-4b9f-9de0-37051403a7c7.png)
I dont know if logs would help ..... maybe there must be some issue in VM
